pkg/controller/node/cidr_set.go

```diff
@@ -38,10 +38,16 @@ type cidrSet struct {
 func newCIDRSet(clusterCIDR *net.IPNet, subNetMaskSize int) *cidrSet {
 	clusterMask := clusterCIDR.Mask
 	clusterMaskSize, _ := clusterMask.Size()
-	maxCIDRs := 1 << uint32(subNetMaskSize-clusterMaskSize)
+
+	var maxCIDRs int
```

Comment:
I will look at that. I also updated the special notes section, this v6 code assumes the subnets be at least /64  (maximum 64 bits of prefix)
The reason for this is described in 
https://github.com/containernetworking/cni/pull/394#issuecomment-287090075